web-components/src/components/cards/ProjectCard/ProjectCard.tsx

```diff
@@ -305,7 +332,7 @@ export function ProjectCard({
                             isPrefinanceProject={isPrefinanceProject}
                             buttonText={buttonText}
                             buttonStartIcon={buttonStartIcon}
-                            buttonClassName={buttonClassName}
+                            buttonClassName={`${buttonClassName} h-full`}
```

Comment:
let's use `cn` function instead please

Reply:
Updated.

I forgot to use `cn` because it seems to work fine without using it. It'd help if you could explain why this is important please.

Reply:
this is useful when mixing multiple types of styles and safer to use, it ensures we have the same behavior everywhere
just look at the `cn` function documentation and implementation for more details.